_includes/releases/v22.2/v22.2.0-beta.6.md

```diff
@@ -0,0 +1,28 @@
+## v22.2.0-beta.6
+
+Release Date: November 7, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-beta-6-sql-language-changes">SQL language changes</h3>
+
+- Fixed a bug that could cause crashes when parsing malformed [changefeed expressions](../v22.2/cdc-transformations.html). [#90844][#90844]
+- Added the boolean column `implicit_txn` to `crdb_internal.cluster_execution_insights` and `crdb_internal.node_execution_insights`. [#90872][#90872]
+
+<h3 id="v22-2-0-beta-6-bug-fixes">Bug fixes</h3>
+
+- TTL decoding error messages now correctly contain hex-encoded key bytes instead of hex-encoded key pretty-print output. [#90726][#90726]
```

Comment:
"TTL" can link to TTL docs at https://www.cockroachlabs.com/docs/v22.2/row-level-ttl.html